src/ClientSideBlazor.Client/Pages/Logout.razor

```diff
@@ -1,6 +1,6 @@
 @page "/logout"
 @inject IAuthService AuthService
-@inject IUriHelper UriHelper
+@inject NavigationManager UriHelper
```

Comment:
Could you renaming the variable to `navigationManager as well to avoid confusion?
```suggestion
@inject NavigationManager NavigationManager
```